src/Symfony/Bundle/FrameworkBundle/Resources/config/profiling.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\HttpKernel\EventListener\ProfilerListener;
+use Symfony\Component\HttpKernel\Profiler\FileProfilerStorage;
+use Symfony\Component\HttpKernel\Profiler\Profiler;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('profiler', Profiler::class)
+            ->public()
+            ->args([service('profiler.storage'), service('logger')->nullOnInvalid()])
+            ->tag('monolog.logger', ['channel' => 'profiler'])
+
+        ->set('profiler.storage', FileProfilerStorage::class)
+            ->args([param('profiler.storage.dsn')])
```

Comment:
please, update and rebase with master branch to get the missing `param` function:
https://travis-ci.org/github/symfony/symfony/jobs/697135256#L3705

Reply:
Done :].